Fan.html

```diff
@@ -34,7 +34,7 @@
             <a class="nav-link" href="bio.html">Bio</a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="Fan.html">Fans</a>
+            <a class="nav-link" href="Fan.html">Fans Page</a>
```

Comment:
Why did you add "Page" in the nav link name? None of the other links have more than one word nor the word "Page" in them...